configs/nextflow.hopper.config

```diff
@@ -0,0 +1,170 @@
+// Hopper specific configuration for micro wgs pipeline //
+
+singularity{
+  enabled = true
+  runOptions = '--bind /fs1/ --bind /local/'
+}
+
+process {
+	executor='slurm'
+	queue='grace-high'
```

Comment:
grace-normal is the correct default queue.

Reply:
Yeah I changed it to grace high because the RNA guys submitted hundreds of jobs and it was taking me forever to run two samples. Will change it back!

Reply:
For dev work you can either have an additional config or just spec it on the command line as well